ReactCommon/jsi/JSCRuntime.cpp

```diff
@@ -184,7 +185,8 @@ class JSCRuntime : public jsi::Runtime {
   static JSStringRef stringRef(const jsi::String& str);
   static JSStringRef stringRef(const jsi::PropNameID& sym);
   static JSObjectRef objectRef(const jsi::Object& obj);
-
+  static JSObjectRef objectRef(const jsi::WeakObject& obj);
```

Comment:
guard this defn with `#ifdef FABRIC_ENABLED`

Reply:
okay cool- I will do that.